vyper/ast/nodes.py

```diff
@@ -934,7 +934,9 @@ def evaluate(self) -> ExprNode:
 
 
 class Operator(VyperNode):
-    pass
+    @property
+    def _op(self):
+        return self._op
```

Comment:
this is going to result in a stack overflow / infinite loop, unless a child class overrides it